.gitignore

```diff
@@ -1,3 +1,4 @@
 **/target
 **/pkg
 **/dist
+Cargo.lock
```

Comment:
1. There is a missing newline. It's mostly just annoying to see that red circle icon here but some tools have problems with it unfortunately. I recommend to just enable "trailing newlines" or how it's called in your IDE.
2. I get used to leave `Cargo.lock` in the repo because then  it's easier to debug broken dependencies but it really doesn't matter.

Reply:
trailing new line was added.
`Cargo.lock` for the application still exist but not for the crate itself.